packages/features/bookings/lib/handleSeats/handleSeats.ts

```diff
@@ -17,7 +17,6 @@ const handleSeats = async (newSeatedBookingObject: NewSeatedBookingObject) => {
     eventType,
     reqBodyUser,
     rescheduleUid,
-    reqBookingUid,
```

Comment:
we are using reqBookingUid below: https://github.com/calcom/cal.com/pull/13406/files#diff-c5f121133ff58fad7910d8fb79c72ea3157cf8d2be2522dadf16fa39894c84aeR41